tubular/scripts/frontend_utils.py

```diff
@@ -127,6 +153,23 @@ def create_version_file(self):
         except IOError:
             self.FAIL(1, 'Could not write to version file for app {}.'.format(self.app_name))
 
+    def copy_js_config_file_to_app_root(self, app_config, app_name):
+        """ Creates a copy of env.config.js(x) file to the app root """
+        source = app_config.get('JS_CONFIG_FILEPATH')
+
+        # Skip if JS_CONFIG_FILEPATH not defined
+        if not source:
+            return
+
+        filename = app_config.get('LOCAL_JS_CONFIG_FILENAME', "env.config.js")
```

Comment:
nit: is it worth having one extra check on line 161 if `app_config.get('LOCAL_JS_CONFIG_FILENAME')` does not exist? Though it should exist since you're defining it in a common location applicable to all MFEs, an MFE could override it to be an empty config settings.